### PR TITLE
Add edge-19.12.1 changes

### PR DESCRIPTION
**Pull request by kleimkuhler**
## edge-19.12.1

* CLI
  * Added condition to the `linkerd stat` command that requires a window size
    of at least 15 seconds to work properly with Prometheus
* Web UI
  * Fixed a table wrap issue in the resource detail view that made sidebar 
    font size inconsistent
* Internal
  * Fixed whitespace path handling in non-docker build scripts (thanks
    @joakimr-axis!)
  * Removed calico logutils dependency that was incompatible with go 1.13
  * Updated Helm templates to use fully-qualified variable references based
    upon Helm best practices (thanks @javaducky!)
  * Added new browser tests for URL routing in dashboard

Signed-off-by: Kevin Leimkuhler <kleimkuhler@icloud.com>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+* CLI
+  * Added condition to the `linkerd stat` command that requires a window size
+    of at least 15 seconds to work properly with Prometheus
+* Web UI
+  * Fixed a table wrap issue in the resource detail view that made sidebar 
+    font size inconsistent
+* Internal
+  * Fixed whitespace path handling in non-docker build scripts (thanks
+    @joakimr-axis!)
+  * Removed calico logutils dependency that was incompatible with go 1.13
```
Comment (alpeb) [inline CHANGES.md]: "Calico"

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+  * Updated Helm templates to use fully-qualified variable references based
+    upon Helm best practices (thanks @javaducky!)
+  * Added new browser tests for URL routing in dashboard
```
Comment (alpeb) [inline CHANGES.md]: _the_ dashboard?

Comment (scottcarol) [inline CHANGES.md]: We don't need to call this (or the React upgrade) out for users

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
+## edge-19.12.1
+
+* CLI
+  * Added condition to the `linkerd stat` command that requires a window size
+    of at least 15 seconds to work properly with Prometheus
+* Web UI
+  * Fixed a table wrap issue in the resource detail view that made sidebar
```
Comment (scottcarol) [inline CHANGES.md]: This is a pretty minor change so not necessary to call out

File: CHANGES.md (hunk 1 above)
Comment (scottcarol) [inline CHANGES.md]: Go is capitalized

**Pull request merged**